src/ACF/Field_Section.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Libs\ACF;
+
+class Field_Section {
```

Comment:
Should `Field_Section` implement the `ACF_Aggregate` interface and/or extend `Field`?